gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -234,7 +234,7 @@
                 <div class="col-md-6 form-group">
                   <label translate>Permissions</label><br />
                   <input type="checkbox" ng-model="currentFolder.ignorePerms" /> <span translate>Ignore</span>
-                  <p translate class="help-block">File permission bits are ignored when looking for changes. Use on FAT file systems.</p>
+                  <p translate class="help-block">Access permissions are ignored when looking for changes. Useful on systems without (e.g. FAT) or incompatible permissions (e.g. synology).</p>
```

Comment:
Access permissions is ambiguous of access to what, it's file permissions.
The "useful" makes it suggestive, where as "use" made it more instructional 

`systems with (ex. FAT)` looks like a cliff hanger.

Use on file systems with non-standard permissions (e.g FAT, synology)